examples/ivr/playRecording.js

```diff
@@ -0,0 +1,24 @@
+// play ivr menu
+const ivrMenu = (conn) => {
+  try {
+    conn.execute('playback', '/usr/share/freeswitch/sounds/en/us/callie/misc/48000/transfer1.wav');
```

Comment:
Anyway you can make this folder location easier to configure? So people who want to use this example don't have to change this path in all locations here?